apps/mobile/src/components/action-bar/action-bar.tsx

```diff
@@ -84,22 +85,21 @@ export const ActionBar = forwardRef<ActionBarMethods, ActionBarProps>(function (
         containerAnimatedStyle,
       ]}
     >
-      <Box width="100%" px="5" justifyContent="center" alignItems="center">
+      <Box
+        borderRadius="sm"
+        overflow="hidden"
+        borderColor="ink.border-transparent"
+        borderWidth={1}
+        style={{ marginHorizontal: theme.spacing[5] - 1 }}
+      >
         <BlurView
+          experimentalBlurMethod="dimezisBlurView"
           themeVariant={themeDerivedFromThemePreference}
-          intensity={90}
+          intensity={Platform.OS === 'ios' ? 95 : 80}
```

Comment:
Is this to get to a fairly similar level of blur? Ought this be abstracted to a value so it can be reused outside this component?

Reply:
I thought of this, but there doesn't seem to be a common denominator between these, so it's just an eyeballed value for this component. I could extract it into `getPlatformSpecificActionBarBlurIntensity` to imply the intention better, until we look into Skia for all our filtering/canvas needs.